test/images/agnhost/netexec/netexec.go

```diff
@@ -112,6 +115,7 @@ func init() {
 		"File containing an x509 private key matching --tls-cert-file")
 	CmdNetexec.Flags().IntVar(&udpPort, "udp-port", 8081, "UDP Listen Port")
 	CmdNetexec.Flags().IntVar(&sctpPort, "sctp-port", -1, "SCTP Listen Port")
+	CmdNetexec.Flags().StringVar(&httpOverride, "http-override", "", "Override the HTTP handler to always respond as if it were a GET with this path & params")
```

Comment:
Could you also add some information about this flag in the agnhost README file? The usage should get updated in the readme as well.

Reply:
Done. I also noticed the TLS flags were missing from the usage string and added them there.